docs/feature_digitizer.md

```diff
@@ -0,0 +1,34 @@
+## Digitizer
```

Comment:
Could you add the functions used for the feature, as well as provide a simple example?

Reply:
Yeah ,I forgot to update the documentation to match the new API. It'll be updated with the next commit.